src/components/Group.js

```diff
@@ -27,12 +27,13 @@ export default function Group(props) {
     element,
     entries = [],
     id,
-    label
+    label,
+    initiallyOpen = false,
```

Comment:
Why not `shouldOpen` as we have in [other components](https://github.com/bpmn-io/properties-panel/blob/main/src/components/entries/Collapsible.js#L25)?

Reply:
We should also make sure to update our [type docs](https://github.com/bpmn-io/properties-panel/blob/main/src/PropertiesPanel.js#L66) (although we don't validate with them by now).